pydoctor/test/test_commandline.py

```diff
@@ -205,3 +221,26 @@ def test_main_source_outside_basedir(capsys: CapSys) -> None:
             'pydoctor/test/testpackages/basic/'
             ])
     assert "Source path lies outside base directory:" in capsys.readouterr().err
+
+
+def test_make_intersphix(tmp_path: Path) -> None:
+    """
+    --make-interphinx will only produce the Sphinx inventory object.
```

Comment:
This happens only when `--make-html` is not passed. I don't think we need a separate test for that, but it might be good to mention here in the docstring.